mapserver.h

```diff
@@ -2807,7 +2807,7 @@ extern "C" {
   MS_DLL_EXPORT outputFormatObj *msSelectOutputFormat( mapObj *map, const char *imagetype );
   MS_DLL_EXPORT void msApplyOutputFormat( outputFormatObj **target, outputFormatObj *format, int transparent, int interlaced, int imagequality );
   MS_DLL_EXPORT const char *msGetOutputFormatOption( outputFormatObj *format, const char *optionkey, const char *defaultresult );
-  MS_DLL_EXPORT outputFormatObj *msCreateDefaultOutputFormat( mapObj *map, const char *driver, const char *name );
+  MS_DLL_EXPORT outputFormatObj *msCreateDefaultOutputFormat( mapObj *map, const char *driver, const char *name, const char *mimetype );
```

Comment:
I'm not sure if this is the best solution to change the prototype of this function. I wanted to be able to get the mimetype from the default output format definition somehow. The only other way to accomplish the change I wanted was to match based on output format name here:

https://github.com/ravhed/mapserver/blob/fb57a5527255fb7b7dd00e8e0363205937ced87e/mapoutput.c#L221

I assume I will need to update the SWIG bindings if this is the way to go.

Reply:
> I assume I will need to update the SWIG bindings if this is the way to go.

Perhaps add a msCreateDefaultOutputFormatEx() variant that takes the extra mimetype parameter. No need to map that to SWIG

Reply:
actually I misunderstood your message. You probably meant that msCreateDefaultOutputFormat() is used by the SWIG bindings. So yes, maybe the simplest is to update them to use the new signature than creating a msCreateDefaultOutputFormatEx() variant

Reply:
This should be addressed now.